node/src/chain_spec/turing.rs

```diff
@@ -183,23 +183,43 @@ fn testnet_genesis(
 		vesting: VestingConfig { vesting_schedule },
 		xcmp_handler: XcmpHandlerConfig { chain_data: xcmp_handler_data },
 		asset_registry: AssetRegistryConfig {
-			assets: vec![(
-				0,
-				orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
-					&orml_asset_registry::AssetMetadata {
-						decimals: TOKEN_DECIMALS,
-						name: "Native".as_bytes().to_vec(),
-						symbol: TOKEN_SYMBOL.as_bytes().to_vec(),
-						existential_deposit: 100_000_000,
-						location: Some(V1(MultiLocation { parents: 0, interior: Here })),
-						additional: CustomMetadata {
-							fee_per_second: Some(416_000_000_000),
-							conversion_rate: None,
+			assets: vec![
+				(
+					0,
+					orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
+						&orml_asset_registry::AssetMetadata {
+							decimals: TOKEN_DECIMALS,
+							name: "Native".as_bytes().to_vec(),
+							symbol: TOKEN_SYMBOL.as_bytes().to_vec(),
+							existential_deposit: 100_000_000,
+							location: Some(V1(MultiLocation { parents: 0, interior: Here })),
+							additional: CustomMetadata {
+								fee_per_second: Some(416_000_000_000),
+								conversion_rate: None,
+							},
 						},
-					},
+					),
+				),
+				(
+					1,
+					orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
+						&orml_asset_registry::AssetMetadata {
+							decimals: 18,
+							name: b"Shibuya".to_vec(),
+							symbol: b"SBY".to_vec(),
```

Comment:
hmm what token symbol to add here is a little tricky. 

First, we are not testing aginst Shibuya, but Rocstar.
Rococo - Rocstar - RSTR
Kusama - Shiden - SDN

Second, is it okay that we use the symbol SDN here, and map it to RSTR for Staging and actual SDN on Turing?

I’ll defer that to Nikhil.

Reply:
Since we're adding tokens that are used exclusively for `development` here, `additonal_assets` should be added as a parameter for to this function `testnet_genesis` and be passed down from the caller. That way we can be explicit about tokens for dev, turing-staging, turing. The `Native` token can stay embedded here since all chains will have the same one.

> Second, is it okay that we use the symbol SDN here, and map it to RSTR for Staging and actual SDN on Turing?

If the same token was used for `turing-staging` and `turing` then we could use a migration here. Since they both have different token symbols, it will need to be a governance proposal. We will need to be careful to make sure that both tokens end up with the same currency id since the integer value of the currency id is referenced in the code.

Reply:
Also, we should check if `Rocstar` and `Shiden` have the same parachain id. If not, it may make more sense to treat them distinctly.

Reply:
Good call. No, they have different paraId, so we will need to treat them differently.
Rocstar: 2006
Shiden: 2007